misc/package/Data Files/MWSE/core/mcm/components/settings/KeyBinder.lua

```diff
@@ -26,11 +26,7 @@ KeyBinder.allowMouse = false
 --- @param keyCode integer|nil
 --- @return string|nil letter
 function KeyBinder:getLetter(keyCode)
-	local letter = table.find(tes3.scanCode, keyCode)
-	local returnString = tes3.scanCodeToNumber[keyCode] or letter
-	if returnString then
-		return string.upper(returnString)
-	end
+	return tes3.getKeyName(keyCode)
 end
```

Comment:
This needs `if not keyCode then return end`. In testing this (with some logging added I got this kind of error:
```
[.\core\mcm\components\settings\keybinder.lua:42] string.format("keyCode: %s, variable: %s", keyCode, inspect(self.variable.value)) = keyCode: false, variable: {
  isAltDown = false,
  isControlDown = false,
  isShiftDown = false,
  keyCode = false,
  mouseButton = 2,
  mouseWheel = false,
  <metatable> = {
    __jsontype = "object"
  }
}
Error in mod config create callback: ...ta Files\MWSE\core\mcm\components\settings\keybinder.lua:30: bad argument #2 to 'format' (number expected, got boolean)
stack traceback:
	.\Data Files\MWSE\core\mcm\main.lua:158: in function <.\Data Files\MWSE\core\mcm\main.lua:128>
```
We store `false` into the keyCode for `KeyBinder` variables when the action is bound to the mouse. 

In the future, it might be the cleanest approach to remove the `getLetter` method entirely and just use `tes3.getKeyName` in `getComboString`. Since I am touching that method in #468, I can change that there, since that PR will need to be updated after we merge yours.

Reply:
Good catch! I'll add a check in `getLetter`. I think you make a good point about the future of the `getLetter` method. It's had a good run.